libraries/AP_Button/AP_Button.cpp

```diff
@@ -183,6 +183,10 @@ void AP_Button::update(void)
             continue;
         }
         const uint16_t pwm_us = pwm_pin_source[i].get_pwm_us();
+        if (pwm_us < 800 || pwm_us > 2200) {
```

Comment:
should be a define, RC_MIN_LIMIT, but should change from 900 to 800

Reply:
done